src/pacmod3_node.cpp

```diff
@@ -610,6 +623,8 @@ int main(int argc, char *argv[])
         kvp.key = "Steering";
       else if (system->first == TurnSignalRptMsg::CAN_ID)
         kvp.key = "Turn Signals";
+      else if (system->first == DoorSignalRptMsg::CAN_ID)
+        kvp.key = "Door Signals";
```

Comment:
"Door" or "Door Status" rather than "Door Signals"

Reply:
We had already DoorRptMsg class, so I just wanted to name it something different

Reply:
Maybe something like "DoorSystemRpt" instead. Signal doesn't really fit.

Reply:
@sanazfth - I just looked at the DBC changes that we made. This system was added as "REAR_PASS_DOOR." Please change to RearPassDoorRpt, RearPassDoorCmd, etc. to match the DBC.

Reply:
Sure, I can do that. But sending command to a "Sliding_Door" makes more sense than sending command to "Rear_pass_door".

Reply:
Depends on if the rear passenger door is automated or not. Regardless, this was the agreed-upon name that is now in the DBC so we should stick with it.